offscreen/scripts/ledger.ts

```diff
@@ -96,7 +96,7 @@ function setupMessageListeners(iframe: HTMLIFrameElement) {
 export default async function init() {
   return new Promise<void>((resolve) => {
     const iframe = document.createElement('iframe');
-    iframe.src = 'https://metamask.github.io/ledger-iframe-bridge/8.0.0/';
+    iframe.src = 'https://metamask.github.io/ledger-iframe-bridge/8.0.3/';
```

Comment:
For the next reviewer: The bridge version has to be aligned with the `@metamask/eth-ledger-bridge-keyring` package version.